state/upgrades.go

```
@@ -1940,6 +1945,9 @@ func updateKubernetesStorageConfig(st *State) error {
 	}
 
 	defaults, err := st.controllerInheritedConfig(model.Cloud())()
+	if err != nil {
```

Good pick up! I am not sure why anyone would want to swallow an error :(